e2e/fablo-config-hlf2-2orgs-2chaincodes-private-data.yaml.test.ts

```diff
@@ -5,3 +5,11 @@ const config = "samples/fablo-config-hlf2-2orgs-2chaincodes-private-data.yaml";
 describe(config, () => {
   performTests(config);
 });
+
+const duplicateChaincodeConfig = "samples/fablo-config-hlf2-2orgs-2chaincodes-private-data-duplicate.yaml";
```

Comment:
I think you should neither update this file nor the yaml.raft.test

Instead it should be a separate test in `fabloCommands.test.ts`, right after: `should print validation errors`, and pretty similar to it